vmm/sandbox/src/stratovirt/config.rs

```diff
@@ -330,7 +355,10 @@ mod tests {
             "-kernel",
             "/var/lib/kuasar/vmlinux.bin",
             "-append",
+            #[cfg(target_arch = "aarch64")]
             "console=hvc0 console=hvc1 iommu=off debug panic=1 pcie_ports=native root=/dev/vda1 ro rootfstype=ext4",
+            #[cfg(target_arch = "x86_64")]
+            "console=hvc0 console=hvc1 iommu=off debug panic=1 pcie_ports=native root=/dev/vda ro rootfstype=ext4",
```

Comment:
Is it possiable to reuse `ROOTFS_KERNEL_PARAMS` insteadof `cfg` again?

Reply:
fixed